packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -359,3 +368,57 @@ export type GPCRevealedClaims = {
    */
   watermark?: PODValue;
 };
+
+/**
+ * General-purpose circuit requirements for a given proof configuration.
+ * These values will be appropriately checked against the circuit configurations
```

Comment:
checked against the _circuits_ at our disposal in order to accommodate all of the proof _requirements_

As above, my goal is to avoid multiple incompatible uses of the words "configuration" and "input".